doc/refman/RefMan-tac.tex

```diff
@@ -3703,6 +3717,9 @@ \subsection{\tt auto}
   This variant is very useful for getting a better understanding of automation,
   or to know what lemmas/assumptions were used.
 
+\item {\tt debug auto} Behaves like {\tt auto} but shows the tactics
```

Comment:
how does it relate to ` info`, which IIRC is what people were using?

Reply:
Feel free to ignore the remark if this is written somewhere else, I don't have the time to read everything.

Reply:
>how does it relate to info, which IIRC is what people were using?

I don't know, I'm not much of a `auto` user.

Reply:
`debug` shows more, it also shows the dead branches, whereas `info` shows just the successful branch.